GOVERNANCE.md

```diff
@@ -0,0 +1,194 @@
+project in discussion with its Collaborators.
+
+## Project nominations
+
+Projects that are widely depended upon within the ESLint ecosystem may be nominated to
+become ESLint-Community projects by anyone by opening a public issue [on GitHub](https://github.com/eslint-community/governance).
+The project should fulfill a few criterias:
+
+The project should:
+
+* Preferably be: An ESLint plugin or formatter
+* Maybe be: A dependency used by an ESLint-Community project or the main ESLint repo
+* Not be: Primarily an ESLint config
+
+The project should also:
+
+* be widely depended upon throughout the community (eg. 3M downloads/week or similar)
+* have an OSI-approved license (preferably same as ESLint itself: MIT)
+* have proposed Collaborators or have possibility to be deemed a Primary project
+  by the Core Team
+
+If project currentlu has an active maintainer it should be transfered and then:
+
+* the current maintainer should be willing to transfer the project to ESLint-Community
+
+If its an abandoned project that needs to be forked:
+
+* It shouldn't have received an update in a substantial amount of time (eg. last 6 months)
+  or it should present a major obstacle to the adoption of eg. a new major version of ESLint
+* A clear notice about the nomination should have been made to the project and its
+  maintainer at least 30 days before it can be accepted
+
+Projects should typically start out with a status of "auxiliary".
+
+## Collaborator nominations
+
+Individuals making significant and valuable contributions to a project may be
+a candidate to join the ESLint-Community organization and that team.
+
+An existing Collaborator needs to open a private team discussion [on GitHub](https://github.com/eslint-community/collaborators/discussions) and
+list the candidates they want to sponsor with a link to the user's contributions. For
+example:
+
+* Activities in the ESLint-Community organization
+  `[USERNAME](https://github.com/search?q=author:USERNAME+org:eslint-community)`
+
+Otherwise, a Contributor may self-apply if they believe they meet the above
+criteria by reaching out to a Core Team member privately with the links to their
+valuable contributions. That Lead Maintainer will then open a private team discussion on
+GitHub regarding that and reply back when a decision has been made.
+
+The consensus to grant a new candidate Collaborator status is reached when:
+
+- at least two of the Core Team members approve
+- at least half of the current collaborators in that team approve, rounded up
+  (and not counting Core Team members)
+
+## Core Team nominations
+
+A Team Member may be promoted to the Core Team only through nomination by a
+Core Team member and with agreement from the rest of Core Team as well as the
+ESLint TSC.
+
+The ESLint TSC can appoint and revoke Core Team members at their own discretion.
+
+## Consensus seeking process
+
+The ESLint Community follows a Lazy [Consensus Seeking]() decision-making model,
```

Comment:
URLs missing?

Reply:
Fixed, I messed up when I was copying from https://github.com/fastify/.github/blob/main/GOVERNANCE.md (which in turn is based on OpenJSF apparently)